Aardvark/ARKEmailBugReporter.h

```diff
@@ -53,6 +54,20 @@
 @end
 
 
+typedef void (^ARKEmailBugReporterCustomPromptCompletionBlock)(ARKEmailBugReportConfiguration *_Nullable configuration);
+
+@protocol ARKEmailBugReporterPromptingDelegate <NSObject>
```

Comment:
The `<NSObject>` specifier may be noise here. Not a strong opinion though since it's idiomatic.

Reply:
Looks like every other protocol in Aardvark does this, so just as well to match.